### ISSUE TITLE
Refined storage compatibility

### ISSUE THREAD

**Issue by c0rn3j** [maruohon/itemscroller#4]
There is no way to take items from the Grid by CTRL/SHIFT.
When holding CTRL/SHIFT and using the Grid's crafting window, it works but also scrolls items in the grid which is pretty annoying.

**Comment by maruohon** [comment]
Could you be a bit more elaborate? I'm not super familiar with Refined Storage.
Are you saying that Item Scroller's functionality doesn't work with Refined Storage inventories (which is a known issue, since they use fake slots), or that Item Scroller breaks something about the Refined Storage inventories?

**Comment by c0rn3j** [comment]
>Are you saying that Item Scroller's functionality doesn't work with Refined Storage inventories (which is a known issue, since they use fake slots)

Yup.

**Comment by maruohon** [comment]
Yep, so unfortunately there is no way for me to know which inventory the slots are in because they are fake slots, and trying to do stuff with them usually just breaks things... There was a crash related to fake slots a while back when I didn't yet check if the slots are "real" and actually in the Container's slot list.
I could talk to raoul at some point to see if there is something that could be done for compatibility, but I think the only "clean" solution (from my end at least) would be that Refined Storage would switch to using actual slots, which I have no idea about how problematic that would be for them.

**Comment by PapiDimmi** [comment]
What are “fake slots”?

Refined Storage compatibility would be great!

**Comment by maruohon** [comment]
Does RS not have their own mouse actions on their GUIs/terminals? AE2 has their own mouse shortcuts.

Fake slots are basically slots that aren't regular normal slots in the Container's slot list. Which means they are handled differently by the mod, and they often don't use the same type of backing inventories as regular slots do. That's why I can't really use any such non-regular slots in Item Scroller, because they don't behave the same way and they may not have functional or standard-compliant inventories tied to them via the vanilla fields/getter methods.
